examples/libtest/libtest.c

```diff
@@ -24,6 +24,7 @@
 
 #include <nuttx/config.h>
```

Comment:
Not sure why for we need include `config.h` here as no check of configuration is done in the file

Reply:
Replaced with <nuttx/compiler.h> for `FAR`.